integration/docker/mem_test.go

```diff
@@ -37,4 +48,74 @@ var _ = Describe("memory constraints", func() {
 			Expect(stderr).To(ContainSubstring("fatal error: runtime: out of memory"))
 		})
 	})
+
+	Context("run container using memory constraints", func() {
+		It("should have applied the constraints", func() {
+			// 512MB
+			memSize = "536870912"
```

Comment:
Nit: Might be worth making this "self-documenting". Something like:

```
mb := 1024 * 1024
memSize = 512 * mb
```

Reply:
done